packit_service/worker/helpers/build/build_helper.py

```diff
@@ -608,3 +616,17 @@ def run_build(
         :return: task_id, task_url
         """
         raise NotImplementedError()
+
+    def report_status_to_configured_job(
+        self,
+        description: str,
+        state: BaseCommitStatus,
+        url: str = "",
+        markdown_content: str = None,
+    ):
+        self.report_status_to_build(
+            description=description,
+            state=state,
+            url=url,
+            markdown_content=markdown_content,
+        )
```

Comment:
What about this?
(A bit dirty, I know..;)


```suggestion
    report_status_to_configured_job = report_status_to_build
```

Reply:
with this, I was not able to test with flexmock that the inherited methods are called :( do you have any hints for that?

Reply:
Aha, ok... If you need to really check which one was called, leave it as is... (You can maybe check the call one step below but that might be too complicated...